tests/output/sql/test_utils.js

```diff
@@ -86,14 +94,6 @@ var assertTest = function(options) {
                     .to.be.equal(options.reason);
             } else {
                 expect(errors).to.not.equal([]);
-                if (options.jshint) {
-                    expect(errors[0].lint).to.be.ok();
-                    expect(errors[0].lint.reason)
-                        .to.be.equal(options.reason);
-                } else {
-                    var $html = $("<div>" + errors[0].text + "</div>");
-                    expect($html.text()).to.be.equal(options.reason);
-                }
```

Comment:
I'm glad this can go away now, but I don't understand why it was needed in this file in the first place since it's used by the sql tests.

Reply:
Generally the test_utils files have a lot in common with each other across pjs/webpage/sql, so I imagine the original was copy/pasta and we didn't notice that there was unused code. (If we had a coverage tool, we might identify such areas)